app/views/groups/new.html.erb

```diff
@@ -1,15 +1,23 @@
 
 <% content_for :title, "CircuitVerse - New Group" %>
-
-<div class="container text-center">
-  <h1 class="">NEW GROUP</h1>
-  <div class="profile-background">
-    <div>
-      Groups can be used by mentors to set projects for and give grades to students.
-      You can find the documentation <a href="https://docs.circuitverse.org/#/groups" target="_blank">here</a>.
+<div class="container">
+  <div class="row row-text">
+    <div class="col-xs col-sm col-md col-lg">
+      <h2 class="heading-submain">NEW GROUP</h2> <br>
+      <h5>Groups can be used by mentors to set projects for and give grades to students.
```

Comment:
This is not a heading but a description so better we use <p> tag here with apt styling (don't make it look too bold)

Reply:
Done